Version 1.0 - Pre check

msevestre opened the pull request.

msevestre commented:
@Yuri05 
![image](https://github.com/Open-Systems-Pharmacology/Qualification-HI/assets/1041237/03e5aa0a-9479-4648-9284-caeefe24dfdb)

Can we disable this?

msevestre commented:
![image](https://github.com/Open-Systems-Pharmacology/Qualification-HI/assets/1041237/da7c4dae-b65c-41fd-bae7-c91bbc588fd7)

Looking for a file that's not even there?

Yuri05 commented:
Please adjust some paths in GH-actions (mostly replace "Evaluation" with "Qualification"):
* https://github.com/Open-Systems-Pharmacology/Qualification-HI/blob/278bca1f08dc909a1325efeb9f295d20b130db3a/.github/workflows/CrossRefCheckWithAnchors.yml#L15
* https://github.com/Open-Systems-Pharmacology/Qualification-HI/blob/278bca1f08dc909a1325efeb9f295d20b130db3a/.github/workflows/MarkdownLinksCheck.yml#L15
* https://github.com/Open-Systems-Pharmacology/Qualification-HI/blob/278bca1f08dc909a1325efeb9f295d20b130db3a/.github/workflows/ValidateEvaluationPlan.yml#L13

Yuri05 commented on Qualification/Input/Content/Section1_Introduction.md:
```diff
@@ -0,0 +1,17 @@
+| Ascites                | None     | Slight   | Moderate |
+| Encephalopathy         | None     | Minimal  | Advanced |
+
+Quantitative anatomical and physiological changes observed in hepatically impaired individuals have an impact on drug PK. These modifications include changes in blood flows (decreased portal and renal blood flow, and incrased hepatic arterial blood flow), cardiac output (increased), plasma binding protein concentrations (decreased), hematocrit (decreased), hepatic intrinsic clearance, and renal i ...
```
Typo: incrased

Yuri05 commented on Qualification/Input/Content/Section2_Model.md:
```diff
@@ -0,0 +1,44 @@
+
+## Blood flow scaling
+
+Portal, hepatic arterial, and renal blood flows were scaled according to the disease factor reported for each Child-Pugh severity ([Table 2](#table_2)). All three blood flow processes were adjustd in PK-Sim by defining Individual > Anatomy & Physiology > Physiology > Flow Rates > Blood Flow Rates. Portal blood flow in the HI population was defined by the Portal Vein specific blood flow rate times ...
```
Typo: `Volums`

Yuri05 replied:
Typo: `adjustd`

Yuri05 commented on Qualification/Input/Content/Alfentanil_introduction.md:
```diff
@@ -0,0 +1,17 @@
+Alfenatanil is a potent, synthetic opioid used to provide analgesia for various surgical procedures and in patients admitted within intensive care units. Alfentanil is typically administered intravenously and offers several advantages to alternative analgesics due to its potency at the opioid receptor and quick onset of effect (3).
```
Typo: `Alfenatanil`

Yuri05 commented on Qualification/Input/Content/Alfentanil_HI.md:
```diff
@@ -0,0 +1,12 @@
+| Ferrier, Marty (10) | 0.05 mg/kg IV bolus | HI population: European females (45%) and males with estimated Child-Pugh B and C | 11 | 56 [range: 39-69] | 60 [range: 43-73] |
+| Ferrier, Marty (10) | 0.05 mg/kg IV bolus | Control population: European | 10 | 45 [range: 25-66] | 59 [range: 34-84] |
+
+Simulated PK of alfentanil after a single IV bolus dose of 0.05 mg/kg in patients with Child-Pugh B and C are presented in **Figure 1** and **Figure 2**, repectively. Simulations of the average healthy control are included for comparison.
```
Typo: `repectively`

Yuri05 commented on Qualification/Input/Content/Levetiracetam_introduction.md:
```diff
@@ -0,0 +1,17 @@
+Levetiracetam is an antiepileptic used for the treatment of patients with partial seizures, with or without secondary generalization (22, 23). The mechanism of antiseizure activity has not been fully elucidated, however, animal models have shown that levetiracetam binds to synaptic vescle proteins SV2A modulating neurotransporter release (24).
```
Typo? should it be ves**i**cle instead of vescle ?

Yuri05 commented on Qualification/Input/Content/Lidocaine_introduction.md:
```diff
@@ -0,0 +1,17 @@
+Levetiracetam is an antiepileptic used for the treatment of patients with partial seizures, with or without secondary generalization (22, 23). The mechanism of antiseizure activity has not been fully elucidated, however, animal models have shown that levetiracetam binds to synaptic vescle proteins SV2A modulating neurotransporter release (24).
```
Typo? should it be ves**i**cle instead of vescle ?

Yuri05 commented on Qualification/Input/Content/Levetiracetam_healthy.md:
```diff
@@ -0,0 +1,26 @@
+| Specific intestinal permeability (transcellular) | 1.84E-3 cm/min |
+| **Oral absorption parameters**             |                             |
+| Formulation: Tablet T<sub>50%</sub>, T<sub>90%</sub>, T<sub>100%</sub> | 5 min, 10 min, 15 min |
+T<sub>50%</sub>, T<sub>90%</sub>, T<sub>100%</sub>: Time to disolve 50%, 90%, and 100% tablet strength, respectively (25).
```
Typo: `disolve`

Yuri05 commented on Qualification/Input/Content/Lidocaine_healthy.md:
```diff
@@ -0,0 +1,26 @@
+| Specific intestinal permeability (transcellular) | 1.84E-3 cm/min |
+| **Oral absorption parameters**             |                             |
+| Formulation: Tablet T<sub>50%</sub>, T<sub>90%</sub>, T<sub>100%</sub> | 5 min, 10 min, 15 min |
+T<sub>50%</sub>, T<sub>90%</sub>, T<sub>100%</sub>: Time to disolve 50%, 90%, and 100% tablet strength, respectively (25).
```
Typo: `disolve`

Yuri05 commented on Qualification/Input/Content/Metoprolol_healthy.md:
```diff
@@ -0,0 +1,29 @@
+
+#### Oral model
+
+The healthy PBPK model development was based on multiple oral PK studies described in the OSP metoprolol report (30) and by Rüdesheim, Wojtyniak (31). Absorption parameters in the oral model included metoprolol tartate tablet described by a Weibull distribution with dissolution half-tim of 12.31 min and shape 0.72.
```
Typo? should it be tart**r**ate instead of tartate ?

Yuri05 replied:
Typo: `half-tim`

Yuri05 commented on Qualification/Input/Content/Theophylline_HI.md:
```diff
@@ -0,0 +1,16 @@
+|--|--|--|--|--|--|
+| Froomes, Morgan (67) | 3 mg/kg IV infusion over 10 min | HI population: European females (40%) and males with Child-Pugh B and C | B: 3; C: 7 | 47.5 [range: 39-64] | Not reported |
+
+Froomes, Morgan (67) observed versus simulated clarance and half-life of theophylline after a 3 mg/kg IV infusion over 10 min in patients with Child-Pugh B and C are presented in **Table 3**. 
```
Typo: `clarance`

Yuri05 commented:
**qualification_plan.json** is invalid:
![grafik](https://github.com/Open-Systems-Pharmacology/Qualification-HI/assets/25061876/dd807c02-d65e-49fd-968a-39b14cfc835b)

Yuri05 commented:
General remark: it is always better to insert URLs as **Markdown-links** and not just as text.
E.g. `[https://www.google.com/](https://www.google.com/)` instead of just `https://www.google.com/ `

When entered as text:
* Links are not properly validated by link-checkers
* Links are sometimes not clickable when converting markdown to other formats via pandoc

Yuri05 commented:
Theophylline population plots in the section 3.6 do not contain any observed data?

Yuri05 commented:
Lidocaine in a HI Population (section 3.7.2) does not contain any plots at all?

Yuri05 commented:
Reference to the Omeprazol OSP model in the section 4 lacks the link to GitHub and "cited.." part
![grafik](https://github.com/Open-Systems-Pharmacology/Qualification-HI/assets/25061876/f398ba1f-5bf7-4036-97f8-0d2da6ea1eb7)

E.g. compare references to other OSP models:
![grafik](https://github.com/Open-Systems-Pharmacology/Qualification-HI/assets/25061876/7406553d-f060-4bda-926e-1ff417257c70)

msevestre commented:
ok I have fixed almost everything. No idea why plots are not available for Lidocaine. They are in the snapshot file and they are also added to the section. Goijng to run again. As for Theophyline without obs data, no idea. Going to ask. Happy to remove it from the report for now

msevestre commented:
@Yuri05 No idea what this error is about

![image](https://github.com/Open-Systems-Pharmacology/Qualification-HI/assets/1041237/6953bd2e-8528-4bb9-8f8a-d5c10ae3ef61)

the file is located at the right place
Going to merge this. we can fix this in another PR

Yuri05 commented:
> @Yuri05 No idea what this error is about
> 
> ![image](https://private-user-images.githubusercontent.com/1041237/341531855-6953bd2e-8528-4bb9-8f8a-d5c10ae3ef61.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTg5NjYwMjQsIm5iZiI6MTcxODk2NTcyNCwicGF0aCI6Ii8xMDQxMjM3LzM0MTUzMTg1NS02OTUzYmQyZS04NTI4LTRiYjktOGY4YS1kNWMxMGFlM2VmNjEucG5nP1gtQW16LUFsZ29yaXRobT1BV1M0LUhNQUMtU0hBMjU2JlgtQW16LUNyZWRlbnRpYWw9QUtJQVZDT0RZTFNBNTNQUUs0WkElMkYyMDI0MDYyMSUyRnVzLWVhc3QtMSUyRnMzJTJGYXdzNF9yZXF1ZXN0JlgtQW16LURhdGU9MjAyNDA2MjFUMTAyODQ0WiZYLUFtei1FeHBpcmVzPTMwMCZYLUFtei1TaWduYXR1cmU9ZjRhNzZmNmMyYWUzMmJiM2VhNjg5NzdiMTQyMmYzMWY2M2M2OTdlYTc1YjcwMTZlMDQwNDRjMWQ5NjkwZmM1MyZYLUFtei1TaWduZWRIZWFkZXJzPWhvc3QmYWN0b3JfaWQ9MCZrZXlfaWQ9MCZyZXBvX2lkPTAifQ.EvZ6S_k1IQ0vTj5RwjWatK3oAxFa4ZAzz8W-8UPqGq4)
> 
> the file is located at the right place Going to merge this. we can fix this in another PR

Case sensitivity. The subfolder should be `Input` and not `input` (we are under Linux 😉 ).
Will create an issue for that.

Yuri05 commented:
> As for Theophyline without obs data, no idea. Going to ask. Happy to remove it from the report for now

I think we should not remove, but rather add pk-ratio plot(s) in the next version (however they are not yet supported for population simulations anyway).
Created an issue for this: https://github.com/Open-Systems-Pharmacology/Qualification-HI/issues/3

msevestre commented:
yes that's the reason. No data available but PK ratio there.
